platform/testFramework/src/com/intellij/testFramework/fixtures/impl/CodeInsightTestFixtureImpl.java

```diff
@@ -184,7 +184,7 @@ private static void removeDuplicatedRangesForInjected(@NotNull List<HighlightInf
       return i != 0 ? i : o1.getSeverity().myVal - o2.getSeverity().myVal;
```

Comment:
If I'm not mistaken, this part looks formatting-only, can you revert it, please?